app/frontend/src/components/Select.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  FormControl,
+  InputLabel,
+  Select as MuiSelect,
+  SelectProps,
+} from "@material-ui/core";
+import classnames from "classnames";
+import React from "react";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme) => ({
+  formControl: {
+    "& .MuiOutlinedInput-root": {
+      borderRadius: theme.shape.borderRadius * 3,
+    },
+    "& .MuiInputBase-input": {
+      height: "auto",
+    },
+    display: "block",
+  },
+}));
+
+export default function Select({
+  id,
+  children,
+  className,
+  label,
+  variant = "outlined",
+  ...otherProps
+}: SelectProps) {
+  const classes = useStyles();
+  return (
+    <FormControl
+      variant={variant}
+      id={id}
+      className={classnames(className, classes.formControl)}
+      margin="normal"
+    >
+      <InputLabel htmlFor={id}>{label}</InputLabel>
+      <MuiSelect
+        native
```

Comment:
I wonder if it's better to not use the `native` option so we get a consistent look and feel between operating systems and also against our other components

Reply:
"As the user experience can be improved on mobile using the native select of the platform, we allow such pattern."
https://material-ui.com/components/selects/#native-select